bootstrap/src/Pharo30Bootstrap.package/PBBootstrap.class/instance/prepareBootstrap.st

```diff
@@ -6,9 +6,5 @@ prepareBootstrap
 		exportKernelPackageNames;
 		exportAllPackagesIntoMcz;
 		exportMonticelloInStFile;
-		exportMultilingualInStFile;
-		exportSUnitInStFile;
-		exportCoreTestsInStFile;
```

Comment:
Are these not used anymore?

Reply:
They are not used anymore. Now the tests of the kernel are loaded in Hermes format, so the kernel tests on the small image does not depend of Opal-Compailer. 
The Multilingual packages where loaded twice, one in the bootstrap and then with the ST. The ST was needed to perform the initialization. 
I have moved the initialization to the 03-initializeUnicode script.